src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -56,230 +57,39 @@ import UtilsService from './UtilsService';
 const MODULE_NAME = 'ChargingStationService';
 
 export default class ChargingStationService {
+
   public static async handleUpdateChargingStationParams(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
     // Filter
     const filteredRequest = ChargingStationValidatorRest.getInstance().validateChargingStationParametersUpdateReq({ ...req.params, ...req.body });
     // Check the Charging Station
     const chargingStation = await UtilsService.checkAndGetChargingStationAuthorization(
-      req.tenant, req.user, filteredRequest.id, Action.UPDATE, action, null, { withSiteArea: true });
-    // OCPI Charging Station
-    if (!chargingStation.issuer) {
-      throw new AppError({
-        errorCode: HTTPError.GENERAL_ERROR,
-        message: `Charging Station '${chargingStation.id}' not issued by the organization`,
-        module: MODULE_NAME, method: 'handleUpdateChargingStationParams',
-        user: req.user,
-        action
-      });
-    }
-    // Site Area
+      req.tenant, req.user, filteredRequest.id, Action.UPDATE, action, null, { issuer: true, withSiteArea: true });
```

Comment:
Issuer should be provided in the auth filter?
So not necessary here, right?

Reply:
@Ramzay any comments on this one?

Reply:
Weird I removed and double checked that the issuer param is no longer needed or uses the one sent from filtered request